doc/source/reference/arrays.indexing.rst

```diff
@@ -111,9 +111,9 @@ concepts to remember include:
               [5],
               [6]]])
 
-- :const:`Ellipsis` expand to the number of ``:`` objects needed to
-  make a selection tuple of the same length as ``x.ndim``. There may
-  only be a single ellipsis present.
+- :const:`Ellipsis` expands to the number of ``:`` objects needed to
+  make a selection tuple of the same length (not including :const:`newaxis`
```

Comment:
It is probably good, although also not quite correct for advanced indexing. Tuple length is a poor fit for original dimensions in some cases. I am OK with this change, though I am not sure it actually helps with understanding, and I do not like bracket notes much. Another option: Maybe make the first sentence say "to index all dimensions" or similar, then a second one to give the tuple analogy which is mostly correct.

Reply:
Yeah, I like "to index all dimensions" better. I pushed a new version. What do you think?

Reply:
Seems a bit better then what is currently there. Looking at the stuff around it a bit, it seems that the previous point uses the same analogy already. And the "tuple length" is not introduced. Now this is the reference so I am OK with just putting it in.

The indexing documentation could use some love. Such as linking the user docs and reference docs, and just general improvements.

If you feel like doing more great. Otherwise, I am good with putting this in as is as a slight improvement.

Reply:
> If you feel like doing more great. Otherwise, I am good with putting this in as is as a slight improvement.

I'll submit PRs if I come across any more errors, but I don't really have time at the moment to work on improving the docs. I came across this error and thought I'd help make a small fix. Please go ahead and merge the PR if you think it's ready.